src/components/dapp-staking-v2/my-staking/MyDapps.vue

```diff
@@ -13,34 +13,48 @@
           </thead>
           <tbody>
             <template v-if="myStakeInfos && myStakeInfos.length > 0">
-              <tr v-for="t in myStakeInfos" :key="t.address">
-                <td>{{ t.name }}</td>
-                <td>
-                  <token-balance
-                    :balance="ethers.utils.formatEther(t.yourStake.denomAmount.toString())"
-                    :symbol="nativeTokenSymbol"
-                    :decimals="0"
-                  />
-                </td>
-                <td>
-                  <div class="row--manage">
-                    <astar-button
-                      width="97"
-                      height="24"
-                      :disabled="false"
-                      @click="navigateToStake(t.dappAddress)"
-                      >{{ $t('myDapps.add') }}</astar-button
-                    >
-                    <astar-button
-                      width="97"
-                      height="24"
-                      :disabled="false"
-                      @click="showUnbound(t)"
-                      >{{ $t('myDapps.unbond') }}</astar-button
-                    >
-                  </div>
-                </td>
-              </tr>
+              <!-- eslint-disable-next-line vue/no-v-for-template-key -->
+              <template v-for="t in myStakeInfos" :key="t.address">
```

Comment:
Could you share with me why don't just use `div` instead of `template`?

Reply:
the reason why to put `tr` tag in `table` as descendant element. please let me know if other better approach for this.

Reply:
I see.